src/logistration/LoginPage.jsx

```diff
@@ -57,6 +58,8 @@ class LoginPage extends React.Component {
   }
 
   handleInstitutionLogin = () => {
+    sendGAEvent('analytics', 'edx.bi.institution_login_form.toggled', null, { category: 'user-engagement' });
+    sendGAEvent('page', 'login_and_registration', 'institution_login');
```

Comment:
`login_and_registration` is the category and `login`, `register` and `institution_login` etc are the names of these events, I guess we are passing these here and on other places as the opposite.